hirosystems/stacks-blockchain-api/templates/api-writer/statefulset.yaml

```diff
@@ -161,7 +161,9 @@ spec:
               if [ -z "$(ls -A "${DATA_DIR}" | grep -v "lost+found")" ]; then
                 echo "Downloading archive"
                 wget ${ARCHIVE_URL} -O ${DATA_DIR}/stacks-node-events.tsv.gz
-                gzip -d ${DATA_DIR}/stacks-node-events.tsv.gz
+                cp -r parquet_generator ${DATA_DIR}
+                cd ${DATA_DIR}
+                python3 -m parquet_generator --tsv-file ${DATA_DIR}/stacks-node-events.tsv.gz
```

Comment:
Does the input TSV file _need_ to be an archive & compressed for this to work?
If so, that may invalidate the scenario where we need to `export` the events from the DB, then re-import them via the parquet process, since the exported events are not an archive nor compressed.

Reply:
yeah, this will be an issue when export and reimporting from postgres; we probably will have to add. to the export logic to gzip it afterwards.

Reply:
Link to our [discussion](https://hiropbc.slack.com/archives/D03U0J6KE4A/p1700249505244389), I believe this is the only part that still needs to be implemented in this PR.

Reply:
update pushed.  hopefully it make sense